sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/BeamSqlCastTest.java

```diff
@@ -67,35 +68,7 @@ public void testCastToDate() {
 
     PAssert.that(result)
         .containsInAnyOrder(
-            Row.withSchema(resultType).addValues(1, new DateTime(2018, 10, 18, 0, 0)).build());
-
-    pipeline.run();
-  }
-
-  @Test
-  public void testCastToDate2() {
```

Comment:
What's wrong with this one?

Reply:
20181018 means year 20181018. This results in a overflow.